Move docs to go repo (#38)
camunda-community-hub/zeebe-client-go: cmd/zbctl/cli-get-started.md

Gateway version: unavailable
Brokers:
  Broker 0 - zeebe-0.zeebe-broker-service.456637ef-8832-428b-a2a4-82b531b25635-zeebe.svc.cluster.local:26501
    Version: unavailable
    Partition 1 : Leader
    Partition 2 : Leader
```

## Advanced process

Use [this process model](https://docs.camunda.io/assets/files/gettingstarted_quickstart_advanced-3786a5f1c2a32871b2cc0580bb44266f.bpmn) for the tutorial.

![processId](./assets/zeebe-modeler-advanced-process-id.png)

This process includes a service task and an XOR gateway. Select the service task and fill in the properties. Set the **Type** to `test-worker`.

![process](./assets/zeebe-modeler-advanced.png)

The worker will return a JSON object as a result, which is used to decide which path to take.

Now, we can use the JSON object to route your process by filling in the condition expression on the two sequence flows after the XOR gateway.

Use the following conditional expression for the **Pong** sequence flow:

```bash
=result="Pong"
```

Use the following conditional expression for the **else** sequence flow:

```bash
=result!="Pong"
```

![sequenceflows](./assets/zeebe-modeler-advanced-sequence-flows.png)

## Deploy a process

> Now, you can deploy the [process](https://docs.camunda.io/docs/bpmn/apis-tools/gettingstarted_quickstart_advanced.bpmn). Navigate to the folder where you saved your process.

Author: ```suggestion
Now, you can deploy the [process](https://docs.camunda.io/assets/files/gettingstarted_quickstart_advanced-3786a5f1c2a32871b2cc0580bb44266f.bpmn). Navigate to the folder where you saved your process.
```

> ---
> id: cli-get-started
> title: Getting started with the CLI client
> sidebar_label: "Getting started with the CLI client"
> description: "Get started with this tutorial that shows you how to interact with Camunda 8 using the community-supported CLI client and command line interface `zbctl`."
> ---

Reviewer: Can get removed?
```suggestion
```

> In this tutorial, you will learn how to use the [community-supported](https://github.com/camunda-community-hub) `zbctl` CLI client to interact with Camunda 8.

Reviewer: ```suggestion
In this tutorial, you will learn how to use the `zbctl` CLI client to interact with Camunda 8.
```

:::note

> The CLI client doesn't support [multi-tenancy](/self-managed/concepts/multi-tenancy.md) and can only be used when multi-tenancy is disabled.

Reviewer: ```suggestion
The CLI client doesn't support [multi-tenancy](https://docs.camunda.io/docs/self-managed/concepts/multi-tenancy/) and can only be used when multi-tenancy is disabled.
```

:::

## Prerequisites

> - [Camunda 8 account](/guides/create-account.md)

Reviewer: ```suggestion
- [Camunda 8 account](https://docs.camunda.io/docs/guides/create-account.md)
```

> - [Cluster](/guides/create-cluster.md)

Reviewer: ```suggestion
- [Cluster](https://docs.camunda.io/docs/guides/create-cluster.md)
```

> - [Client credentials](/guides/setup-client-connection-credentials.md)

Reviewer: ```suggestion
- [Client credentials](https://docs.camunda.io/docs/guides/setup-client-connection-credentials.md)
```

> - [Modeler](/guides/model-your-first-process.md)

Reviewer: ```suggestion
- [Modeler](https://docs.camunda.io/docs/guides/model-your-first-process.md)
```
